tools/importer-rest-api-specs/components/parser/models.go

```diff
@@ -148,7 +148,7 @@ func (d *SwaggerDefinition) detailsForField(modelName string, propertyName strin
 	}
 
 	// first get the object definition
-	objectDefinition, nestedResult, err := d.parseObjectDefinition(modelName, propertyName, &value, result)
+	objectDefinition, nestedResult, err := d.parseObjectDefinition(modelName, propertyName, &value, result, false)
```

Comment:
It's hard to understand exactly what the bool is doing here, so if we assign that to an intermediate variable it's far clearer:

```suggestion
	parsingModel  := false
	objectDefinition, nestedResult, err := d.parseObjectDefinition(modelName, propertyName, &value, result, parsingModel)
```